client/src/assets/scss/main.scss

```diff
@@ -1,7 +1,25 @@
 @import "./foundation/settings";
 @import "~foundation-sites/scss/foundation";
 @include foundation-everything;
+@import "../../../public/stylesheets/index.css";
+@import url("https://use.typekit.net/qtt8vii.css");
+
 // place global css here
+
+$dark-gray: #545454;
+$light-gray: #A7A8A5;
+$blue: #406BB0;
+$green: #4FA99F;
+$yellow: #EBC539;
+$red: #CB2E2E;
+
+* {
```

Comment:
We really shouldn't be using the asterisk here, but rather apply these styles to either the `body` of the app or a specific element.